pandas/io/gbq.py

```diff
@@ -52,6 +52,7 @@ def _test_google_api_imports():
         except:
             from apiclient.discovery import build  # noqa
             from apiclient.errors import HttpError  # noqa
+        from oauth2client.client import GoogleCredentials  # noqa
```

Comment:
Can this line be moved inside of `try`?

Using oauth2client version 1.2, I receive the following exception:

```
>>> from oauth2client.client import GoogleCredentials
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name GoogleCredentials
```